src/extension/content-script/batteries/Twitter.ts

```diff
@@ -84,9 +88,10 @@ function battery(): void {
         recipient = match[1];
       } else {
         // if we did not find anything let's look for an ⚡ emoji
-        const zapElements = userData.element.querySelectorAll(
-          'img[src*="26a1.svg"]'
-        );
+        const zapElements = new Set([
+          ...userData.element.querySelectorAll('img[src*="26a1.svg"]'),
+...(userData.location?.querySelectorAll('img[src*="26a1.svg"]') || []),
```

Comment:
is there a formatting issue?

```suggestion
        ...(userData.location?.querySelectorAll('img[src*="26a1.svg"]') || []),
```

Reply:
Hmm. I commited from Github, that might have caused it. Will checkout & commit locally.

Reply:
@bumi fixed